package.json

```diff
@@ -36,6 +36,7 @@
   "peerDependencies": {
     "history": "^2.0.0 || ^3.0.0",
     "react": "^0.14.0 || ^15.0.0",
+    "prop-types": "^15.0.0",
```

Comment:
I wasn't sure where to put it, If it's not a peer, should it be in the normal dependencies?